kong/db/schema/entities/plugins.lua

```diff
@@ -8,13 +8,15 @@ return {
   cache_key = { "name", "route", "service", "consumer" },
   dao = "kong.db.dao.plugins",
   workspaceable = true,
+  endpoint_key = "custom_name",
```

Comment:
We should be wary of user input in endpoint keys. It should not be possible to pick a name that conflicts with an entity that a plugin can be configured to. (route, service, etc..)

Reply:
is that a really a problem:
```
/plugins/route/route
```

certainly looks funny, but will work?